core-tests/src/test/java/org/mule/runtime/core/el/context/ExceptionTestCase.java

```diff
@@ -30,15 +31,16 @@ public ExceptionTestCase(Variant variant, String mvelOptimizer) {
   public void exception() throws Exception {
     MuleEvent event = getTestEvent("");
     RuntimeException rte = new RuntimeException();
-    event.setMessage(MuleMessage.builder(event.getMessage()).exceptionPayload(new DefaultExceptionPayload(rte)).build());
+    event.setError(builder(rte).build());
```

Comment:
Same as in the other PR, I think the static imports in the builder() case don't make it clear.